demisto_sdk/commands/content_graph/objects/script.py

```diff
@@ -25,3 +25,11 @@ def prepare_for_upload(self, marketplace: MarketplaceVersions = MarketplaceVersi
             data['nativeImage'] = supported_native_images
 
         return data
+
+    @property
+    def imported_by(self) -> list:
```

Comment:
```suggestion
    def imported_by(self) -> List[Integration]:
```

Reply:
Changed to `List[BaseContent]` instead because that is what RelationshipData keeps:

``` 
class RelationshipData(BaseModel):
    relationship_type: RelationshipType
    source: BaseContent
    target: BaseContent

    # this is the attribute we're interested in when querying
    content_item: BaseContent
```